lib/vicadmin/validate.go

```diff
@@ -180,7 +180,7 @@ func NewValidator(ctx context.Context, vch *config.VirtualContainerHostConfigSpe
 	nwErrors := []error{}
 
 	for _, host := range hosts {
-		conn, err := net.Dial("tcp", host)
+		conn, err := net.DialTimeout("tcp", host, time.Minute/2)
```

Comment:
Who cares what compiler does for this case? I wasn't trying to micro optimize the generated code but instead trying to make the source more readable - at least for myself 30 * time.Seconds is much more easy to parse than time.Minute /2 

Anyhow, it's merged so no big deal.